src/openrct2/interface/Window.h

```diff
@@ -75,20 +75,20 @@ enum class WindowWidgetType : uint8_t;
 
 struct Widget
 {
-    WindowWidgetType type;
-    uint8_t colour;
-    int16_t left;
-    int16_t right;
-    int16_t top;
-    int16_t bottom;
+    WindowWidgetType type{};
+    uint8_t colour{};
+    int16_t left{};
+    int16_t right{};
+    int16_t top{};
+    int16_t bottom{};
     union
     {
-        uint32_t content;
+        uint32_t content{};
         ImageId image;
         StringId text;
         utf8* string;
     };
-    StringId tooltip;
+    StringId tooltip{ 0xFFFF };
```

Comment:
This should use a constant.

Reply:
It could just be `STR_NONE`, defined in StringIdType.h. But then again, this PR is a draft for a reason, I imagine.